### PR TITLE
Add first of Silero STT models

### PR DESCRIPTION
**Pull request by snakers4**
Hi,

My name is Alexander, I am with Silero, we are a small independent self-financed company making speech related products.
Please kindly review our Speech-To-Text models.

I believe if this PR is ok, in future we will follow with other languages and tf-js versions as well.

### CODE REVIEW

**Comment by snakers4** [comment]
@sayakpaul 
Hi Paul, I found that you are quite active here, just a small question - do I need to ask someone from the community to merge the PR?

**Comment by sayakpaul** [comment]
@snakers4. Probably @maringeo will review your PR and will merge it. The community contributors like me don't have the merging privileges I guess.

**Comment by snakers4** [comment]
@sayakpaul 
Many thanks for the heads up!

**Comment by sayakpaul** [comment]
@snakers4 just on a side note do you folks have any plans to release the TensorFlow Lite variant of the models?

**Comment by snakers4** [comment]
> @snakers4 just on a side note do you folks have any plans to release the TensorFlow Lite variant of the models?

we mostly do research using pytorch and we used tf for its tf-js capabilities (i.e. pytorch => onnx => tf => tfjs)
I have not yet thought about tf-lite and it use-cases as in pytorch fusing / quantization / export is done in the main library and more real-life mobile applications would probably require models about 10MB (now we have 50MB)

but any contributions are welcome ofc in silero [models](https://github.com/snakers4/silero-models)!

**Comment by sayakpaul** [comment]
@khanhlvg FYI.

File: tfhub_dev/assets/silero/models/silero-stt/en/1.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Module silero/silero-stt/en/1
+A set of compact enterprise-grade pre-trained STT Models for multiple languages.
+
+<!-- asset-path: https://silero-models.ams3.cdn.digitaloceanspaces.com/models/en/en_v1_tf.tar -->
```
Comment (maringeo) [inline tfhub_dev/assets/silero/models/silero-stt/en/1.md:4]: Is it possible to compress (`gzip`) the `en_v1_tf.tar` asset to `en_v1_tf.tar.gz`?

**Comment by maringeo** [comment]
Hi @snakers4,

Since this is your first time contributing to the TFhub.dev repository, we just need you to read & agree to the TensorFlow Hub terms of service at https://tfhub.dev/terms. If you feel comfortable moving forward, please add a comment "I have read and agree to the TensorFlow Hub Terms of Service" to this thread.

File: tfhub_dev/assets/silero/models/silero-stt/en/1.md (hunk 1 above)
Comment (snakers4) [reply tfhub_dev/assets/silero/models/silero-stt/en/1.md:4]: Yes, ofc, I will do it later today

**Comment by snakers4** [comment]
> Since this is your first time contributing to the TFhub.dev repository, we just need you to read & agree to the TensorFlow Hub terms of service at https://tfhub.dev/terms. If you feel comfortable moving forward, please add a comment "I have read and agree to the TensorFlow Hub Terms of Service" to this thread.


"I have read and agree to the TensorFlow Hub Terms of Service"

File: tfhub_dev/assets/silero/models/silero-stt/en/1.md (hunk 1 above)
Comment (snakers4) [reply tfhub_dev/assets/silero/models/silero-stt/en/1.md]: @maringeo 
I re-uploaded the tgz file and changed the link in the PR

**Comment by maringeo** [comment]
Hi @snakers4 , this PR has been merged in 02ab9b7d3455e99e97abecf43c5d598a5528e20c and it's live on tfhub.dev:

- publisher page: https://tfhub.dev/silero
- collection page: https://tfhub.dev/silero/collections/silero-stt/1
- model page: https://tfhub.dev/silero/silero-stt/en/1

**Comment by snakers4** [comment]
@maringeo 

Looks like our model does not show when I go here
image

Looks like audio is just a set of tags, can you add `audio-stt` there as well?
Many thanks!

**Comment by maringeo** [comment]
Hi @snakers4, good catch, I'll add `audio-stt` to the set of tags. It should be live next week.

**Pull request closed without merging**